vcr/api/v1/api.go

```diff
@@ -135,16 +136,24 @@ func (w *Wrapper) Create(ctx echo.Context) error {
 	return ctx.JSON(http.StatusOK, vcCreated)
 }
 
-// Resolve a VC and return its content
-func (w *Wrapper) Resolve(ctx echo.Context, id string) error {
+func (w *Wrapper) Resolve(ctx echo.Context, id string, params ResolveParams) error {
 	idURI, err := ssi.ParseURI(id)
 	// return 400 for malformed input
 	if err != nil {
 		return core.InvalidInputError("failed to parse credential ID: %w", err)
 	}
 
+	// resolve time
+	at := time.Now()
+	if params.At != nil {
+		at, err = time.Parse(time.RFC3339, *params.At)
```

Comment:
I'm wondering, what's the use-case for using another resolve time (in the past for example)?

Reply:
mostly for when things go wrong. If an access token requests fails for invalid credentials, you would be able to redo the request with the same timestamp.

Reply:
Check!